src/api/article/localized/utils.js

```diff
@@ -0,0 +1,44 @@
+import isEmpty from 'lodash/isEmpty';
+
+import { ValidationError } from 'utils/validation';
+import { defaultValidator } from 'utils/joi';
+import { mapIds } from 'utils/getters';
+
+import LocalizedArticle, { joiLocalizedArticleSchema } from './model';
+
+const validate = locales => {
+  const errors = locales.reduce((acc, cur) => {
+    const err = defaultValidator(cur, joiLocalizedArticleSchema);
+    if (err) {
+      acc[cur.locale] = err;
+    }
+    return acc;
+  }, {});
+  if (!isEmpty(errors)) {
+    throw new ValidationError({ locales: errors });
+  }
+};
+
+const update = locales =>
+  Promise.all(
+    // Q: why not `findOneAndUpdate`?
```

Comment:
who are you addressing the question here?

Reply:
you and me :)